### PR TITLE
test/run-cli-tests: use cram from github

### PR DESCRIPTION
**Pull request by tchaikov**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tchaikov** [comment]
@liewegas if you are fine with this solution. I will fork cram into ceph/cram.

**Comment by tchaikov** [comment]
jenkins test make check

**Comment by tchaikov** [comment]
jenkins test make check

File: src/test/run-cli-tests [hunk 1]
```diff
@@ -30,7 +30,7 @@ if [ ! -e "$CRAM_BIN" ]; then
     # patched cram to support that. See upstream ticket at
     # https://bitbucket.org/brodie/cram/issue/9/allow-read-only-directories-for-t
     # -- tv@inktank.com
-    virtualenv "$VENV" && $VENV/bin/pip --log "$VENV"/log.txt install "$SRCDIR/downloads/cram-0.5.0ceph.2011-01-14.tar.gz"
+    virtualenv --python python3 "$VENV" && $VENV/bin/pip --log "$VENV"/log.txt install git+https://github.com/tchaikov/cram.git#egg=cram
```
Comment (sebastian-philipp) [inline src/test/run-cli-tests:33]: want to add a tag to the url? otherwise you can't push to the repo, without directly affecting the runs

Comment (tchaikov) [reply src/test/run-cli-tests]: @sebastian-philipp thanks! now using "0.7-error-dir".

**Review by liewegas — APPROVED** [review]

**Comment by tchaikov** [comment]
```
 73/215 Test  #13: run-cli-tests .............................   Passed  176.92 sec
```

**Comment by smithfarm** [comment]
@tchaikov Does this need to be backported to `{pacific,octopus,nautilus}`? I noticed cram-related rbd-cli-tests failures in pacific PRs - e.g. https://github.com/ceph/ceph/pull/39048

**Comment by smithfarm** [comment]
backport to pacific now ongoing in https://github.com/ceph/ceph/pull/39048

**Comment by tchaikov** [comment]
@smithfarm  yes, I think so. thanks for creating the octopus backport!

**Pull request merged**